ui/components/TracePage/TracePage.tsx

```diff
@@ -832,14 +842,17 @@ export const TracePage = () => {
 
   return !traceId ? null : (
     <TreeProvider key={traceId} traceId={traceId}>
-      <Head>
-        <title>
-          {traceId && recipes[traceId]
-            ? `${recipes[traceId].title} | Interactive Composition Explorer`
-            : "Interactive Composition Explorer"}
-        </title>
-      </Head>
-      <Trace traceId={traceId} />
+      <PromptEditorProvider>
+        <Head>
+          <title>
+            {traceId && recipes[traceId]
+              ? `${recipes[traceId].title} | Interactive Composition Explorer`
+              : "Interactive Composition Explorer"}
+          </title>
```

Comment:
Might be worth extracting <title> into a component

Reply:
We can probably do that at a later date, but it's out of scope with this PR